centralized_settings.json

```diff
@@ -0,0 +1,24 @@
+{
+	"pause_all": false,
+	"pause_periods": [
+	  {
+		"name": "woo_update_6_1",
+		"start": "2024-02-16 00:00:00",
```

Comment:
Could I make an argument for using [ISO 8601](https://en.wikipedia.org/wiki/ISO_8601) as the date format? 😄 It's as unambiguous as it gets -- instantly recognizable, guaranteed to be parsed correctly by `strtotime`, and no question as to which timezone the string represents.

Reply:
So, would that be changing `2024-02-16 00:00:00` to `2024-02-16T00:00:00+00:00` for example?

Or, setting them all to U.S. East coast timezone using the `-04:00` suffix?

Reply:
Yes, exactly! Sorry, I should've included those examples myself 😅 PHP calls [this format ATOM](https://www.php.net/manual/en/class.datetimeinterface.php#datetimeinterface.constants.atom) which I think is pretty cool too;

Reply:
If we get a timestamp from the setting, we can simply generate that string doing: 

```
$time = new DateTime();
$time->setTimestamp( 1709548419 );

echo $time->format( DateTime::ATOM );
```

That would output `2024-03-04T10:33:39+00:00` right now.